pkg/sso/client.go

```diff
@@ -87,11 +91,18 @@ func (c *Client) GetAuthorizationURL(opts GetAuthorizationURLOptions) (*url.URL,
 	c.once.Do(c.init)
 
 	query := make(url.Values, 5)
-	query.Set("domain", opts.Domain)
 	query.Set("client_id", c.ProjectID)
 	query.Set("redirect_uri", c.RedirectURI)
 	query.Set("response_type", "code")
 
+	if opts.Domain == "" && opts.Provider == "" {
+		return nil, errors.New("Incomplete arguments. Need to specify either a 'domain' or 'provider'")
```

Comment:
Should not use capital letters in error messages.
=> https://github.com/golang/go/wiki/Errors

```
errors.New("incomplete arguments: missing domain or provider")
```